site/docs/concepts/materialization.md

```diff
@@ -317,6 +317,17 @@ A given endpoint may support standard updates, delta updates, or both.
 This depends on the [materialization connector](../reference/Connectors/materialization-connectors/README.md). Expect that a connector will use
 standard updates only unless otherwise noted in its documentation.
 
+### Delta updates for every binding in a Materialization
+
+When configuring a materialization, you can set delta updates as the materialziation mechanism
+for every binding. Afterwards, you are able to modify this setting individually for each binding
+for further customization.
+
+:::note
+This functionality is not supported for all Materialization connectors. Consult the individual connector
+documentation for details.
```

Comment:
Can we be more specific in this note and in the schema name note on what to look for? Something like: "This functionality is only supported for materialization connectors with [xyz]. Consult the individual connector documentation for details."

Reply:
Clarified a bit, let me know if it makes sense